README.md

```diff
@@ -10,6 +10,7 @@ Most use cases for micropayments involve a single user or device interacting
 repeatedly with a few vendors.
 
 ## Prerequisites
+`Iota-swarm-node` is compatible with Python 3.5.
```

Comment:
The wording is misleading. Instead, python 3.5+ is required as [PyOTA](https://github.com/iotaledger/iota.lib.py) configures. You have to emphasize on the rationale in plain English descriptions, otherwise nobody can be aware of it.